gradle.properties

```diff
@@ -0,0 +1,8 @@
+mlAppName=JSdebugTestServer
+mlHost=localhost
+mlUsername=admin
+mlPassword=changeme-in-gradle-local.properties
+mlRestPort=8055
+mlContentDatabaseName=Documents

```

Comment:
I am hoping it's an anti-pattern that Documents/Modules are used, and that was just done before so a developer didn't have to create these databases as well. Can you try removing this and the modules property below so that you get new content/modules databases? That should simplify things and avoid any issues with what someone already has in those databases.

Reply:
Hmm - I just did a search on `'Modules` and saw that some tests refer to that. Looks like it should be simple to change that to `JSdebugTestServer-modules` - though as long as we're changing this, let's use "mlxprs-test" as the name instead so someone can look at the Admin UI and have an idea of what the purpose of the app server is.